processor/error/package_tests/TestProcessErrorNullValues.approved.json

```diff
@@ -3,6 +3,7 @@
         {
             "@timestamp": "2017-05-09T15:04:05.999999Z",
             "context": {
+                "process": {},
```

Comment:
I think `process` should be able to be `null`, so this should be checked here. 
If `process` is given, it should however have a `pid`, at least this is required in the spec above. If you remove the `null` type in the spec, sending an empty hash for `process` should not be possible any longer.